README.md

```diff
@@ -19,3 +19,23 @@ This repository is a monorepo of the various components required for Chainlink o
 
     # Then run
     asdf install
+
+# Running e2e tests
```

Comment:
Could you please add the links you shared in the slack channel like where to go and what to do in order to configure AWS access